state/tstate/recorder_test.go

```diff
@@ -0,0 +1,165 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package tstate
+
+import (
+	"context"
+	"crypto/sha256"
+	"testing"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/hypersdk/keys"
+)
+
+func FuzzRecorderPermissionValidator(f *testing.F) {
+	for i := 0; i < 10; i++ {
+		shaBytes := sha256.Sum256([]byte{byte(i), byte(i >> 8)})
+		bytes := shaBytes[:]
+		for bytes[len(bytes)-1] > 16 {
+			shaBytes := sha256.Sum256([]byte{bytes[len(bytes)-2], bytes[len(bytes)-1]})
+			bytes = append(bytes, shaBytes[:]...)
+		}
+		f.Add(bytes)
+	}
+	f.Fuzz(
+		RecorderPermissionValidatorFuzzer,
+	)
+}
+
+func createKeys(prefix byte) (keylist [][]byte) {
```

Comment:
nit: could we remove the naked returns and named return variables throughout? We generally avoid this throughout the codebase

Reply:
done